code/src/App.js

```diff
@@ -1,9 +1,78 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+
+import Form from './components/Form'
+import Map from './components/Map'
+import Hero from './components/Hero'
+
+import { API_URL, LIKES_URL } from './reusable/urls'
 
 export const App = () => {
+  const [messageList, setMessageList] = useState([]) 
+  const [messageNew, setMessageNew] = useState("") // The state that takes the input from our submit button
+
+  useEffect(() => {
+    fecthMessageList()
+
+  }, [])
+
+  const fecthMessageList = () => {
+    fetch(API_URL)
+    .then(res => res.json())
+    .then(message => setMessageList(message))
+    .catch(err => console.error(err))
+  }
+
+  const OnInputMessage = (event) => { 
+    setMessageNew(event.target.value)
+  }
+
```

Comment:
A small comment here: 
the function names should use camel case and should not start with capital letter. So it should be onInputMessage